src/dapp/components/farm/Gold.tsx

```diff
@@ -125,6 +125,32 @@ export const Gold: React.FC<Props> = ({ inventory }) => {
           !machineState.matches("mining") &&
           (isNextToChop || isHighlighted);
 
+        const waitingBoundLimiterContainerStyle = {
```

Comment:
Where possible try to create a sibling css file and import that.

E.g. `Gold.css` 

In the future we are looking to use JSS stylesheets with Typescript support but for now we do it the old fashion way with CSS 😅

Reply:
Yeah, that makes sense @adamhannigan – can do!